pos_partner_deselection/README.rst

```diff
@@ -2,11 +2,11 @@
    :target: https://www.gnu.org/licenses/lgpl
    :alt: License: LGPL-3
 
-==================
- Deselect Partner
-==================
+====================
+Partner  deselection
+====================
 
-POS Partner deselection after a defined time interval
+The module allows to deselect partner in opened POSes by the user after a defined time interval.
```

Comment:
`by the user` is incorrect. Users by default are able to deselect customers. Main Idea of the module is that customer is deselected **Automatically**
Smth like:
The module deselects a partner in opened POS after a defined time interval after the customer was set
But in proper english

Reply:
The module deselects a partner in opened POS on expiry the predefined time interval after the customer was set.